packages/web-components/src/components/card-link/__stories__/card-link.stories.ts

```diff
@@ -17,10 +17,15 @@ import textNullable from '../../../../.storybook/knob-text-nullable';
 import '../index';
 
 export const Default = ({ parameters }) => {
-  const { disabled, href } = parameters?.props?.CardLink ?? {};
+  const { disabled, href, copy } = parameters?.props?.CardLink ?? {};
   return html`
     <dds-card-link ?disabled=${disabled} href=${ifNonNull(href || undefined)}>
-      Explore AI use cases in all industries
+      IBM developer
+      ${copy
+        ? html`
+            <p>${copy}</p>
+          `
+        : ``}
```

Comment:
I think we need to define a `copy` slot within this HTML return that allows for easy styling if needed.

`<slot name="copy"></slot>`

With this slot, you can include the following markup to the Storybook preview.

`<dds-card-link-copy slot="copy">text</dds-card-link-copy>`

@annawen1 or @ariellalgilmore Let me know if you both agree.

Reply:
Thanks @proeung and @mwanberg! Since DDSCardLink is extending DDSCard I think we actually want to keep the copy like @mwanberg implemented, but I think it would make sense to actually change the `heading` to use the `heading slot` and apply the heading styles appropriately there. Currently both the copy and heading are being rendered within the `copy div`

cc: @annawen1 

![image](https://user-images.githubusercontent.com/20210594/129076830-92b0f6f9-b2b3-4da8-ac0a-1800f24c6c01.png)

Reply:
I agree with @ariellalgilmore, we should utilize the `heading` slot

Reply:
Makes sense. Thank you both! 

@mwanberg I would leverage the existing card heading component and adjust the type styling to use `expressive-heading-02`  within the `_card-link.scss` partial as defined in the design specs (see link -  https://ibm.ent.box.com/file/657497664059?s=pkjve2gmmldfi9dinak8ihahkct9hb9f).